Remove unnecessary 'grep' qualifiers in 'unit-tests'

We need to change our `grep` implementation in our `unit-tests` as follows:

```bash
## check unit test results
ERRORS=$(grep '= FAILURES =|= ERRORS =|= no tests ran in 0.00 seconds =' pytest.log)
if [ -n "$ERRORS" ]; then
    echo 'Error: unit test exited.'
    exit 1
fi
```

--- a/test/unit-tests
+++ b/test/unit-tests
@@ -64,7 +64,7 @@ sudo docker run --name database --net=app_nw -d container-database && \
 sudo docker run --name webserver-pytest --net=app_nw -it container-webserver test | sudo tee pytest.log
 
 ## check unit test results
-ERRORS=$(! grep -E '= FAILURES =|= ERRORS =|= no tests ran in 0.00 seconds =' pytest.log)
+ERRORS=$(grep '= FAILURES =|= ERRORS =|= no tests ran in 0.00 seconds =' pytest.log)
 if [ -n "$ERRORS" ]; then
     echo 'Error: unit test exited.'
     exit 1

